docs/docsite/rst/dev_guide/testing_integration.rst

```diff
@@ -98,9 +98,9 @@ Tests in Docker containers
 If you have a Linux system with Docker installed, running integration tests using the same Docker containers used by
 the Ansible continuous integration (CI) system is recommended.
 
-.. note: Docker on non-Linux::
+.. note:: Docker on non-Linux
 
-   Using Docker Engine to run Docker on a non-Linux host is not recommended.
+   Using Docker Engine to run Docker on a non-Linux (such as macOS) host is not recommended.
```

Comment:
Please revise to "Using Docker Engine to run Docker on a non-Linux host (such as macOS) is not recommended.